scripts/build.sh

```diff
@@ -340,19 +372,17 @@ if [ ! -d libdivecomputer/src ] ; then
 	git submodule update --recursive
 fi
 
-cd libdivecomputer
-
-mkdir -p build
-cd build
+mkdir -p "${BUILD_PREFIX}libdivecomputer/build"
+cd "${BUILD_PREFIX}libdivecomputer/build"
 
-if [ ! -f ../configure ] ; then
+if [ ! -f $SRC/subsurface/libdivecomputer/configure ] ; then
```

Comment:
Quote the argument to [ here. This will break if $SRC contains any "funny" chars or a " " for example.